packages/utils/src/misc.ts

```diff
@@ -58,6 +58,21 @@ export function getEnvironment(): string {
   return ENV_MAP.unknown;
 }
 
+export function getBundleId(): string | undefined {
+  try {
+    if (isReactNative() && typeof global !== "undefined" && (global as any)?.Application) {
+      if (typeof (global as any).Application?.getConstants === "function") {
+        return (global as any).Application?.getConstants().applicationId;
+      } else if (typeof (global as any).Application?.applicationId === "string") {
+        return (global as any).Application?.applicationId;
+      }
```

Comment:
```suggestion
return (global as any).applicationId
```
isn't it cleaner to just assign the applicationId to the `global` and move these conditions in the `react-native-compat`? This way all of the RN logic/conditions would be isolated to `react-native-compat`

Reply:
Yes it is cleaner, but i thought that maybe is also easier for malicious devs to override the appId in react-native-compat code? Maybe im overthinking it.